build.gradle

```diff
@@ -35,21 +35,58 @@ buildscript {
 }
 
 ext.versions = [
+        akkaHttpVersion : '10.2.7',
+        akkaVersion : '2.6.15',
+        archaiusVersion: "latest.release",
+        avroVersion : "1.10.2",
+        cliParserVersion : '1.1.1',
+        configMagicVersion : '0.11',
+        curatorVersion : '2.12.+',
+        fenzoVersion : '0.13.8',
         flink    : "1.14.2",
+        gsonVersion : "2.8.+",
         hadoop   : "2.7.7",
+        hdrHistogramVersion : '2.+',
+        icebergVersion : "0.14.+",
+        jctoolsVersion : "1.+",
+        jodaTimeVersion : '2.+',
+        jsonVersion : '20180813',
         junit4   : "4.11",
         junit5   : "5.4.+",
+        kafkaVersion : "2.2.+",
+        mesosVersion : '1.7.2',
         mockito  : "2.0.+",
         mockito3 : "3.+",
+        nettyVersion : "4.1.34.Final",
+        parquetVersion : "1.12.0",
+        rxJavaReactiveStreamsVersion : '1.+',
+        scalaBinaryVersion : '2.12',
+        snappyVersion : "1.1.+",
         spectator: "1.3.+",
         slf4j    : "1.7.0",
+        testngVersion : '6.14.+',
         vavr     : "0.9.2",
 ]
 
 ext.libraries = [
+        akkaActor      : "com.typesafe.akka:akka-actor_${versions.scalaBinaryVersion}:${versions.akkaVersion}",
+        akkaHttp       : "com.typesafe.akka:akka-http_${versions.scalaBinaryVersion}:${versions.akkaHttpVersion}",
+        akkaHttpCaching: "com.typesafe.akka:akka-http-caching_${versions.scalaBinaryVersion}:${versions.akkaHttpVersion}",
+        akkaHttpJackson: "com.typesafe.akka:akka-http-jackson_${versions.scalaBinaryVersion}:${versions.akkaHttpVersion}",
```

Comment:
I would merge these three deps as akkaHttp since they are used together. Look at the junitJupiter dependency for reference.